debian/source/options

```diff
@@ -0,0 +1,4 @@
+compression = "gzip"
```

Comment:
I think we have a great opportunity to significantly reduce our build times here.

The main reason our .deb (and .rpm) builds are taking a long time is because they try to compress all files. And by all files, I also mean the `container.tar.gz` file, which is already compressed at the highest level. This file is basically responsible for the inflated size of the Debian package, and we don't gain something by re-compressing it.

My suggestion is to add here a `compression-level = "fast"` option (taken from https://manpages.debian.org/stretch/dpkg-dev/dpkg-source.1.en.html#Generic_build_options). I believe the change in size will be minimal (~1-2 MBs), but the change in build time will be massive. If this holds, we can also do something similar for RPM packages in a subsequent PR.

Reply:
:sad-trombone: Unfortunately, it actually doesn't seem to make much difference (on my machine at least) : 

|  | time (real) | time (user) | time (sys) | Package size |
|--------|--------|--------|--------|--------|
| `compression-level = "fast"` | 1m29.722s | 5m52.673s | 0m8.686s | 658M |
| Default value | 1m30.911s | 6m2.769s | 0m8.471s | 658M |

Reply:
They mention it in their docs actually: Note that this option will not cause existing tarballs to be recompressed, it only affects new files

Reply:
Sad trombone indeed. Thanks for taking a look at this Alexis.

_Weird, then what is it compressing for 6 minutes straight..._